src/particles/spacecharge/Gauss3dPush.cpp

```diff
@@ -73,27 +77,35 @@ namespace impactx::particles::spacecharge
             // Simpson's rule integration
 
             // Ex
-            amrex::ParticleReal const fx = f1 / f2x;
+            //amrex::ParticleReal const fx = f1 / f2x;
+            fx = f1 / f2x;
```

Comment:
With the current implementation, the `fx` value would only be the last value of `i=nint`, so one could calculate it outside of the loop.